mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -44,23 +61,31 @@
   private static final String NAME = "name";
   private static final String LOCATION = "location";
 
+  private static final Set<String> PROPERTIES_TO_REMOVE = Stream
+      .of(InputFormatConfig.TABLE_SCHEMA, InputFormatConfig.PARTITION_SPEC, LOCATION, NAME)
+      .collect(Collectors.toCollection(HashSet::new));
```

Comment:
What about `Collectors.toSet`? Doesn't that use a hash set by default?

Reply:
Done